src/deepsparse/transformers/pipelines/text_classification.py

```diff
@@ -86,7 +90,10 @@ class TextClassificationOutput(BaseModel):
 )
 class TextClassificationPipeline(TransformersPipeline):
     """
-    transformers text classification pipeline
+    transformers text classification pipeline.
+    Scores are returned as sigmoid over logits if one label is given or
```

Comment:
Why one label, what's the logic here?

Reply:
sotmax would always return 1 for a single label problem. this additionally matches the transformers logic flow

Reply:
Oh, got it.